code/xcvm/lib/core/README.md

```diff
@@ -0,0 +1,6 @@
+# Overview
+
+Staff to do cross chains apps. 
```

Comment:
```suggestion
Stuff to do cross chains apps. 
```

Reply:
oh, i disabled spell checking as too many false positives. so will fix this